pkg/commands/print.go

```diff
@@ -0,0 +1,48 @@
+package commands
+
+import (
+	"time"
+
+	"github.com/briandowns/spinner"
+	"github.com/fatih/color"
+	"github.com/gosuri/uitable"
+	"github.com/kyokomi/emoji"
+)
+
+// colors used in vela cmd for printing
+var (
+	red    = color.New(color.FgRed)
+	green  = color.New(color.FgGreen)
+	yellow = color.New(color.FgYellow)
+	white  = color.New(color.Bold, color.FgWhite)
+)
+
+// emoji used in vela cmd for printing
+var (
+	emojiSucceed   = emoji.Sprint(":check_mark_button:")
+	emojiFail      = emoji.Sprint(":cross_mark:")
+	emojiLightBulb = emoji.Sprint(":light_bulb:")
+)
+
+// newUITable creates a new table with fixed MaxColWidth
+func newUITable() *uitable.Table {
+	t := uitable.New()
+	t.MaxColWidth = 80
```

Comment:
In most lines of setting MaxColWidth, the value is 60. Why do you change the it to 80? To make the display more readable for `vela traits/workloads`?

Reply:
Yes, currently capability description contains much more content to display.

Reply:
It might be better to set the default MaxColWidth to 60, and only set it to 80 for `vela workloads/traits`. These lines can be removed.

![image](https://user-images.githubusercontent.com/2805315/101312275-4dfefc80-388e-11eb-9980-91ca5ecdfd88.png)